docs/tutorials/using-jsx-in-apostrophe.md

```diff
@@ -312,14 +325,18 @@ function App({ defaultCity }) {
   return (
     <Container>
       <AppLabel>React Weather App</AppLabel>
-      <CityComponent updateCity={updateCity} fetchWeather={handleFetchWeather} />
+      <CityComponent
+        updateCity={updateCity}
```

Comment:
camel case is... standard in react... react is weird

Reply:
React liberally mixes camel, pascal, kebab, and screaming snake.